src/planning.py

```diff
@@ -217,9 +220,15 @@ def task_plan(
 
 
 def _skeleton_generator(
-    task: Task, ground_nsrts: List[_GroundNSRT], init_atoms: Set[GroundAtom],
-    heuristic: _TaskPlanningHeuristic, seed: int, timeout: float,
-    metrics: Metrics, max_skeletons_optimized: int
+    task: Task,
+    ground_nsrts: List[_GroundNSRT],
+    init_atoms: Set[GroundAtom],
+    heuristic: _TaskPlanningHeuristic,
+    seed: int,
+    timeout: float,
+    metrics: Metrics,
+    max_skeletons_optimized: int,
+    abstract_ldl: LiftedDecisionList = None
```

Comment:
let's remove ` = None` here, since it will always be passed down from `sesame_plan`. also, let's change the type to `Optional[LiftedDecisionList]`, since it can be either an LDL or None. (I'm surprised mypy didn't complain about that)